plugin.xml

```diff
@@ -30,8 +30,8 @@
     <repo>https://github.com/apache/cordova-plugin-wkwebview-engine</repo>
 
 	<engines>
-        <engine name="cordova-ios" version=">=4.0.0-dev" />
-        <engine name="apple-ios" version=">=9.0" />
+            <engine name="cordova-ios" version=">=4.0.0 <6.0.0" />
```

Comment:
While updating these.. Might be able to also fix the indention of the entire `<engines>` block. Seems to be indented too much.

Reply:
This is what I get for being lazy and using vim...